src/Microsoft.AspNetCore.Mvc.RazorPages/PageModel.cs

```diff
@@ -1675,7 +1675,7 @@ public virtual void OnPageHandlerExecuting(PageHandlerExecutingContext context)
         }
 
         /// <summary>
-        /// Called after the handler method executes, before the action method is invoked.
+        /// Called after the handler method executes.
```

Comment:
> Called after the handler method executes, before the action result executes

Maybe? I think it's trying to pinpoint where it appears in the filter pipeline.

Reply:
Sure, sound good. Will update